projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/tiled-gallery-rectangular.php

```diff
@@ -312,6 +381,48 @@ public function get_weighted_ratio() {
  * Tiled gallery group class.
  */
 class Jetpack_Tiled_Gallery_Group {
+	/**
+	 * Images to be displayed in group.
+	 *
+	 * @var array
```

Comment:
The constructor's parameter declaration says `object`, but  probably should actually say `object[]`.

Reply:
Done in b503fca7d2b44b8c41de15263072a4145cc1b69c